breakpoints/getCallbackFromUserFriendlyCallbackArgument.js

```diff
@@ -50,7 +50,31 @@ function getTraceFunction(predefinedBreakpoint) {
     else {
         traceFn = function(debugInfo){
             runWithBreakpointsDisabled(function(){
-                console.trace("About to " + debugInfo.accessType + " property '" + debugInfo.propertyName + "' on this object: ", debugInfo.object)
+                if (debugInfo.accessType == "set") {
+                    var newPropertyValue = debugInfo.newPropertyValue;
+                    var newPropertyType = typeof newPropertyValue;
+
+                    if (newPropertyType === "string" || newPropertyType === "number") {
+                        newPropertyValue = newPropertyValue.toString();
+
+                        if (newPropertyValue.length > 10) {
+                            newPropertyValue = newPropertyValue.substring(0, 10) + "...";
+                        }
+
+                        if (newPropertyType === "string") {
+                            newPropertyValue = "'" + newPropertyValue + "'";
+                        }
+                    }
+                    else if (newPropertyType !== "boolean") {
```

Comment:
The reason I didn't include the `boolean` branch was that it was redundant. I have included it now if that helps with clarity of logic for you, but I'm not sure it's good practice.